editor/components/inserter/index.js

```diff
@@ -36,8 +41,12 @@ class Inserter extends wp.element.Component {
 					icon="insert"
 					label={ wp.i18n.__( 'Insert block' ) }
 					onClick={ this.toggle }
-					className="editor-inserter__toggle" />
-				{ opened && <InserterMenu position={ position } onSelect={ this.close } /> }
+					className="editor-inserter__toggle"
+					aria-haspopup="true"
+					id="inserter-toggle"
+					buttonRef={ ( node ) => this.nodes.toggle = node }
```

Comment:
> IconButton component into a class component instead of a functional component to allow ref on it?

If we do that how to we get the ref to the `button` element deep in the tree that needs to be focused.

>We'll need a focus method implemented in the IconButton also I guess.

This sounds like something that could work a bit better pass in a prop of some sort and track the inserter button focus as part of the Inserter's state?